src/fsharp/fsi/fsi.fs

```diff
@@ -1499,17 +1518,21 @@ module internal MagicAssemblyResolution =
         Assembly.LoadFrom(path)
 #endif
 
-    let ResolveAssembly(m,tcConfigB, tcImports: TcImports, fsiDynamicCompiler: FsiDynamicCompiler, fsiConsoleOutput: FsiConsoleOutput, fullAssemName:string) = 
+    let Install(tcConfigB, tcImports: TcImports, fsiDynamicCompiler: FsiDynamicCompiler, fsiConsoleOutput: FsiConsoleOutput) = 
```

Comment:
I'm not sure what this change is doing.  I suspect the FCS version is the right one